.github/workflows/ci.yaml

```diff
@@ -735,6 +737,26 @@ jobs:
           ./scripts/build_docker.sh \
             --arch amd64 \
             --target ghcr.io/coder/coder-preview:main \
+            --target
             --version $version \
             --push \
             build/coder_linux_amd64
+
+          # Get commit sha to be used as package tag
+          new_package_tag=$(gh api /repos/coder/coder/commits/main | jq -r '.sha[:8]')
+
+          # Tag image with new package tag and push
+          docker tag ghcr.io/coder/coder-preview:main ghcr.io/coder/coder-preview:main-$new_package_tag
+          docker push ghcr.io/coder/coder-preview:main-$new_package_tag
+
+      - name: Prune old images
+        uses: vlaurin/action-ghcr-prune@v0.5.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          organization: coder
+          container: coder-preview
+          dry-run: true
+          keep-younger-than: 7 # days
```

Comment:
this will delete all old image tags older than 7 days that does not start with `pr` e.g. `pr1234` and will delete images that start with `main-` i.e. `main-632hst52`

Reply:
Also, `dry-run` is true and we can let it run in dry-run mode for a few days and then switch off when we are confident that it is safe to use.

Reply:
cc: @deansheather

Reply:
this will also do not delete custom pushed tags like `gvisor-fix`